cmd/wksctl/profile/constants/constants.go

```diff
@@ -1,6 +1,7 @@
 package constants
 
 const (
-	AppDevAlias   = "app-dev"
-	AppDevRepoURL = "git@github.com:weaveworks/eks-quickstart-app-dev"
+	AppDevAlias          = "app-dev"
+	AppDevRepoURL        = "git@github.com:weaveworks/eks-quickstart-app-dev"
+	WKSctlIgnoreFilename = ".wksctlignore"
```

Comment:
Please separate the `...Filename` constant from the `AppDev...` constants, and add a comment describing what the new constant means.

Reply:
ack

Reply:
fixed